packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2710,49 +2710,88 @@ class _Location {
   }
 }
 
-/// Describer to gather important information about [offendingContext].
+/// Transformer to parse and gather information about [offendingContext].
 ///
-/// This describer is only applicable when the [offendingContext] is created in
-/// widget layer.
+/// This transformer only responds to [DiagnosticsProperty] with the name
+/// 'offendingContext'.
 ///
-/// This function will be registered to [FlutterErrorDetails.offendingContextDescribers]
+/// This function will be registered to [FlutterErrorDetails.informationTransformers]
 /// in [WidgetsBinding.initInstances].
-Iterable<DiagnosticsNode> describeOffendingContext(dynamic offendingContext) {
+DiagnosticPropertiesBuilder transformOffendingContext(DiagnosticPropertiesBuilder properties) {
```

Comment:
this code should be written so that for the common case that there is no
DebugCreator, the original List<DiagnosticsNode> is returned. Currently a new list is build even if no work needs to be done.